src/components/home/VolunteerBrief.jsx

```diff
@@ -4,110 +4,112 @@ import Button from '@mui/material/Button';
+    icon: <StorageRoundedIcon fontSize="large" />,
     label: 'Data Scientists',
     alt: 'Database'
   },
   {
-    icon: <FaPenRuler size={45} />,
+    icon: <DesignServicesOutlinedIcon fontSize="large" />,
     label: 'Designers',
     alt: 'Pen and ruler'
   },
   {
-    icon: <FaPeopleLine size={45} />,
+    icon: <GroupsOutlinedIcon fontSize="large" />,
     label: 'Project Managers',
     alt: 'People in a line'
   },
   {
-    icon: <FaChartPie size={45} />,
+    icon: <QueryStatsOutlinedIcon fontSize="large" />,
     label: 'Product Managers',
     alt: 'Pie chart'
   },
   {
-    icon: <FaHandshakeSimple size={45} />,
+    icon: <HandshakeOutlinedIcon fontSize="large" />,
     label: 'Social Workers',
     alt: 'Handshake'
   },
   {
-    icon: <FaHandHoldingHeart size={45} />,
+    icon: <VolunteerActivismOutlinedIcon fontSize="large" />,
     label: 'Community Organizers',
     alt: 'Hand holding heart'
   },
   {
-    icon: <FaUser size={45} />,
+    icon: <PersonOutlinedIcon fontSize="large" />,
     label: 'Anyone!',
     alt: 'User avatar'
   }
 ];
 
+const boxStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  textAlign: 'center',
+  width: '100%',
+  m: { xs: '50px 0 100px 0', md: '50px 0 150px 0' }
+};
+
+const cardStyle = {
+  m: { xs: '10px 0 10px 0', md: '0 25px 20px 25px' },
+  p: 4,
+  backgroundColor: 'rgba(217, 217, 217, 0.4)',
+  borderRadius: '30px'
+};
+
 const VolunteerBrief = () => {
   return (
-    <Box
-      as="section"
-      sx={{
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        textAlign: 'center',
-        width: '100%',
-        my: '40px'
-      }}
-    >
-      <Typography variant="h3" component="h2" sx={{ px: '2rem' }}>
+    <Box as="section" sx={boxStyle}>
+      <Typography variant="h3" component="h2" sx={{ mb: '40px' }}>
         Volunteer With Us
       </Typography>
-      <Card
-        sx={{
-          py: 5,
-          my: 5,
-          backgroundColor: 'lightgrey',
-          borderRadius: '10px'
-        }}
-      >
+      <Card sx={cardStyle}>
         <Typography variant="h6" component="h3" sx={{ px: '1rem' }}>
           Use your skills for the greater good or build your resume!
         </Typography>
-        <Grid container rowSpacing={5} sx={{ margin: 0 }}>
-          {volunteerGrid.map(({ icon, label, alt }) => (
-            <Grid item xs={6} sm={3} key={alt}>
-              <Typography variant="body2">
-                {icon ?? null}
-                <br />
-                {label ?? null}
-              </Typography>
-            </Grid>
-          ))}
-        </Grid>
+        <Box>
+          <Grid
+            container
+            rowSpacing={3}
+            sx={{
+              // m: 0,
+              pt: 3
+            }}
+          >
+            {volunteerGrid.map(({ icon, label, alt }) => (
+              <Grid item xs={6} sm={3} key={alt}>
+                <Typography variant="body2">
```

Comment:
seems a little strange to have the `<p>` house the icon...

I would do this, and no longer need the `<br>` either:
```js
{volunteerGrid.map(({ icon, label, alt }) => (
              <Grid item xs={6} sm={3} key={alt}>
                {icon ?? null}
                <Typography variant="body2">{label ?? null}</Typography>
              </Grid>
            ))}        
 ```